src/schema/metadata/AnatomicalLandmarkCoordinates.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name: AnatomicalLandmarkCoordinates
+description: |
+  Key:value pairs of any number of additional anatomical landmarks and their
+  coordinates in voxel units (where first voxel has index 0,0,0)
+  relative to the associated anatomical MRI
+  (for example, `{"AC": [127,119,149], "PC": [128,93,141],
+  "IH": [131,114,206]}`, or `{"NAS": [127,213,139], "LPA": [52,113,96],
+  "RPA": [202,113,91]}`).
+  Each array MUST contain three numeric values corresponding to x, y, and z
+  axis of the coordinate system in that exact order.
```

Comment:
Here we have a problem unfortunately - and I should have already noticed it when reviewing MEG. This field has been defined differently (2 versions) across the spec: 

- once for MRI: using voxel units ([mri](https://bids-specification.readthedocs.io/en/latest/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#anatomical-landmarks))
- once for MEG and EEG: using the origin, axes, and units that the `AnatomicalLandmarkCoordinateSystem` is in ([meg](https://bids-specification.readthedocs.io/en/latest/04-modality-specific-files/02-magnetoencephalography.html#coordinate-system-json-_coordsystemjson), [eeg](https://bids-specification.readthedocs.io/en/latest/04-modality-specific-files/03-electroencephalography.html#coordinate-system-json-_coordsystemjson))

What's common is:

>  Each array MUST contain three numeric values corresponding to x, y, and z axis of the coordinate system in that exact order.

Reply:
We have two ways to deal with different definitions at the moment:

1. Appended text, as you've seen.
2. Completely different YAML files, with different filenames but the same "name" field. 
 
I am reticent to use the second option, because it can easily be abused to make every section's version of a field an entirely different entity. That said, it seems like this could be a reasonable use-case for that. WDYT?

Reply:
>  it can easily be abused to make every section's version of a field an entirely different entity

yeah, that's something we should avoid. If that's restricted to a few minor cases where we are bound by backward compatibility, I think it'd be okay (such as here).

Reply:
Sounds like a plan. This should be handled in c0df0ad. Please let me know what you think.

Reply:
Thanks @tsalo I think this is sensible. Let's hope others agree. I'll continue with my review soon!

PS: No idea why linkchecker keeps failing. the error 104 (connection reset by peer) seems to be an issue of the linkchecker or our testing platform, rather than the remote server (those pages are perfectly well accessible)

Reply:
but why does linkchecker keep failing here but passes on master?

Reply:
I really have no clue. The failures seem to be on fairly random links. Sometimes it's MNI template pages, sometimes it's tedana's documentation, sometimes it's other things...

Reply:
is this branch up to date with `master`? If not, can we manage that? It'd be good to get to the bottom of this before merging this PR.

Reply:
ok, that wasn't it. I restarted the flow. maybe we get lucky. It'd be nice to have a clean link checker run, because after that the pdf job will run and I'd like to check those tables as well. Because in some way the changes here do affect the table formatting :-)

Reply:
linkchecker is fixed, and we have a solution for the original issue as well. I am resolving this